app/src/main/java/net/skyscanner/backpack/demo/SettingsActivity.kt

```diff
@@ -56,15 +56,16 @@ class SettingsActivity : AppCompatActivity() {
     )
 
     val themeToggle = findViewById<BpkSwitch>(R.id.theme_toggle)
+    val themePicker = findViewById<View>(R.id.theme_picker)
 
-    themeToggle?.apply {
-      setOnCheckedChangeListener { _, isChecked ->
-        onThemeSwitchClicked(isChecked)
-      }
+    themePicker.setOnClickListener {
```

Comment:
set the click listener on the container instead to make the whole row being read out in one go to make it clearer what you're toggling